src/app/about/About.module.css

```diff
@@ -4,6 +4,54 @@
     text-align: center;
 }
 
+/* CSS Modules recommends camelCase
+See: https://github.com/css-modules/css-modules#naming */
+.imageContainer {
+    display: flex;
+    justify-content: center;
+    margin-top: 2rem;
+}
+
+.title {
+    margin-bottom: 5rem;
+    font-size: 3rem;
+}
+
+.description {
+    margin: 0 auto;
+    max-width: 1000px;
+}
+
 .image {
     margin: auto;
 }
+
+.officers {
+    /* align */
```

Comment:
[Nit/Please Change]: These comments don't really add anything, especially as this isn't a standard CSS convention for grouping (applies to other files and such too).

Would also say that `display: flex` & `justify-content: center` should be considered alignment and not content, so this kind of grouping gets a little messy.